services/sample_configuration/application.yml

```diff
@@ -60,9 +60,6 @@ spring:
   cloud:
     consul:
       enabled: false
-  # Configure jackson mappers so that null properties are not included in the serialized response.
-  jackson:
-    default-property-include: non_null
```

Comment:
Why is this removed? It was there to support compatibility with prior web service responses.

Reply:
Yikes. No idea. Lol. Inadvertent cut I guess. Adding back

Reply:
It's not critical in the sample configuration, of course. I brought it up since I wanted to make sure there won't be a conflict down the road with actual deployments. Sounds like maybe that's not the case.

Reply:
Restored in latest commit